### PR TITLE
Mvc/cdn network dns

### PR DESCRIPTION
**Pull request by sptramer**

### CODE REVIEW

**Comment by sankethka** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/cdn.md](https://github.com/sptramer/azure-docs-sdk-python/blob/mvc/cdn-network-dns/docs-ref-services/cdn.md) | :white_check_mark:Succeeded | |
[docs-ref-services/dns.md](https://github.com/sptramer/azure-docs-sdk-python/blob/mvc/cdn-network-dns/docs-ref-services/dns.md) | :white_check_mark:Succeeded | |
[docs-ref-services/network.md](https://github.com/sptramer/azure-docs-sdk-python/blob/mvc/cdn-network-dns/docs-ref-services/network.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C19%5Ca92b76e3-3152-c0a9-537b-4d37096a01a1%5CPullRequest%5C201707191914486952-74%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sankethka** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/cdn.md](https://github.com/sptramer/azure-docs-sdk-python/blob/mvc/cdn-network-dns/docs-ref-services/cdn.md) | :white_check_mark:Succeeded | |
[docs-ref-services/dns.md](https://github.com/sptramer/azure-docs-sdk-python/blob/mvc/cdn-network-dns/docs-ref-services/dns.md) | :white_check_mark:Succeeded | |
[docs-ref-services/network.md](https://github.com/sptramer/azure-docs-sdk-python/blob/mvc/cdn-network-dns/docs-ref-services/network.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C19%5Ca92b76e3-3152-c0a9-537b-4d37096a01a1%5CPullRequest%5C201707191928323583-74%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sankethka** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/cdn.md](https://github.com/sptramer/azure-docs-sdk-python/blob/mvc/cdn-network-dns/docs-ref-services/cdn.md) | :white_check_mark:Succeeded | |
[docs-ref-services/dns.md](https://github.com/sptramer/azure-docs-sdk-python/blob/mvc/cdn-network-dns/docs-ref-services/dns.md) | :white_check_mark:Succeeded | |
[docs-ref-services/network.md](https://github.com/sptramer/azure-docs-sdk-python/blob/mvc/cdn-network-dns/docs-ref-services/network.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C19%5Ca92b76e3-3152-c0a9-537b-4d37096a01a1%5CPullRequest%5C201707192120451177-74%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/cdn.md [hunk 1]
```diff
@@ -16,11 +16,49 @@ ms.service: multiple
...
+from azure.mgmt.cdn import CdnManagementClient
+
+cdn_client = CdnManagementClient(credentials, 'your-subscription-id')
+profile = cdn_client.profiles.create('my-resource-group',
+                                     'cdn-name',
+                                     {"location": "some_region", 
+                                      "sku": {
+                                        "name": "sku_tier"} 
+                                    })
+new_endpoint = client.endpoints.create('my-resource-group',
```
Comment (lmazuel) [inline docs-ref-services/cdn.md]: endpoint_poller = cdn_client.endpoints.create('my-resource-group'
..........
new_endpoint = endpoint_poller.result()

File: docs-ref-services/cdn.md [hunk 2]
```diff
@@ -16,11 +16,49 @@ ms.service: multiple
...
+
+### Example
+
+Creating a CDN profile with a single defined endpoint:
+
+```python
+from azure.mgmt.cdn import CdnManagementClient
+
+cdn_client = CdnManagementClient(credentials, 'your-subscription-id')
+profile = cdn_client.profiles.create('my-resource-group',
```
Comment (lmazuel) [inline docs-ref-services/cdn.md]: profile_poller = cdn_client.profiles.create('my-resource-group',
............
profile = profile_poller.result()

File: docs-ref-services/dns.md [hunk 3]
```diff
@@ -16,11 +16,37 @@ ms.service: multiple
...
+Create a new DNS zone.
+
+```python
+from azure.mgmt.dns import DnsManagementClient
+
+dns_client = DnsManagementClient(credentials, 'your-subscription-id')
+zone = dns_client.zones.create_or_update('resource-group',
+                                         'zone_name_no_dot',
+                                         {
+                                            "location": "region"
```
Comment (lmazuel) [inline docs-ref-services/dns.md]: DNS is a global service, the only available location is "global":
```python
        zone = self.dns_client.zones.create_or_update(
            self.group_name,
            account_name,
            {
                'location': 'global'
            }
        )
```

File: docs-ref-services/cdn.md (hunk 2 above)
Comment (sptramer) [inline docs-ref-services/cdn.md]: Fixed a2ce4a3

File: docs-ref-services/cdn.md (hunk 1 above)
Comment (sptramer) [inline docs-ref-services/cdn.md]: Fixed a2ce4a3

File: docs-ref-services/dns.md (hunk 3 above)
Comment (sptramer) [inline docs-ref-services/dns.md]: Fixed a2ce4a3

**Comment by sankethka** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/python-sdk-azure-authenticate.md](https://github.com/sptramer/azure-docs-sdk-python/blob/mvc/cdn-network-dns/docs-ref-conceptual/python-sdk-azure-authenticate.md) | :white_check_mark:Succeeded | |
[docs-ref-services/cdn.md](https://github.com/sptramer/azure-docs-sdk-python/blob/mvc/cdn-network-dns/docs-ref-services/cdn.md) | :white_check_mark:Succeeded | |
[docs-ref-services/dns.md](https://github.com/sptramer/azure-docs-sdk-python/blob/mvc/cdn-network-dns/docs-ref-services/dns.md) | :white_check_mark:Succeeded | |
[docs-ref-services/network.md](https://github.com/sptramer/azure-docs-sdk-python/blob/mvc/cdn-network-dns/docs-ref-services/network.md) | :white_check_mark:Succeeded | |
[docs-ref-services/postgresql.md](https://github.com/sptramer/azure-docs-sdk-python/blob/mvc/cdn-network-dns/docs-ref-services/postgresql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C20%5Ca92b76e3-3152-c0a9-537b-4d37096a01a1%5CPullRequest%5C201707201844110214-74%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**